webui/src/lib/api/index.js

```diff
@@ -670,12 +670,15 @@ export const uploadWithProgress = (url, file, method = 'POST', onProgress = null
             resolve({
                 status: xhr.status,
                 body: xhr.responseText,
-                contentType: xhr.getResponseHeader('Content-Type'),
```

Comment:
From what I can tell this header is indeed not used anywhere. 
Can you validate this?

Reply:
I checked and it is used. Keeping it.

Reply:
index.js line 756. I think it is still needed in case of a regular upload - not to a presigned url.

Reply:
can't find it 😅